Write metatext (#189)
SW709E19/Report: report/ChapterTest/ChapterCtrl.tex

> This chapter starts by introducing the acceptance tests of the system, the results of the acceptance test and the conclusion of them.

Reviewer: Dette skal skrives om, og så skal det skrives i datid og være mere overordnet
```suggestion
This chapter documents the different methods used to test the system. The results of these tests will be elaborated on, and a conclusion formed.
```

> This chapter documents the different methods used to test the system. The results of these tests will be elaborated on, and a conclusion formed.

Reviewer 2: ```suggestion
This chapter documents the tests conducted on the system.
```

> in addition to this, the chapter also documents the quality of service for the system and the guarantees the system can make.

Reviewer 2: ```suggestion
In addition to this, the chapter documents the system's quality of service and guarantees the system can make.
```
